src/Tizen.NUI/src/internal/Transition/TransitionSet.cs

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI
+{
+    using System;
+    using System.ComponentModel;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// TransitionSet is used to control lifetime of multiple Transitions.
+    /// For the one page transition, may multiple transitions are played coincidently.
+    /// Every transitions added on a TransitionSet have same play lifetime. And emit a single Finished signal.
+    /// </summary>
+    /// <since_tizen> 3 </since_tizen>
```

Comment:
Please remove since_tizen tag for hidden API

Reply:
Thank you.